qt_ui/windows/QLiberationWindow.py

```diff
@@ -266,6 +269,16 @@ def saveGameAs(self):
             liberation_install.setup_last_save_file(self.game.savepath)
             liberation_install.save_config()
 
+            self.updateWindowTitle(file)
+
+    def updateWindowTitle(self, file):
+        file_name = (
+            file[0].split("/")[-1].split(".liberation")[0]
+        )  # file is a tuple (path, extension)
+        self.setWindowTitle(
+            f"{self.windowTitle()} - {file_name}"
+        )  # append file name to end of existing title
```

Comment:
If you open files repeatedly this will keep appending. I'm guessing you were just trying to avoid the duplication from `__init__`. Move the version part here since `__init__` calls this too?